tests/storage/datavolume.go

```diff
@@ -536,7 +536,7 @@ var _ = SIGDescribe("DataVolume Integration", func() {
 				vm, err = virtClient.VirtualMachine(vm.Namespace).Create(context.Background(), vm)
 				Expect(err).ToNot(HaveOccurred())
 
-				Eventually(ThisVMIWith(vm.Namespace, vm.Name), 100).Should(BeInPhase(v1.Pending))
+				Eventually(ThisVMIWith(vm.Namespace, vm.Name), 100).Should(Or(BeInPhase(v1.Pending), BeInPhase(v1.Scheduling)))
```

Comment:
I would expect a `Scheduling` VMI to become scheduled at some point, unless there is some scheduling problem. But if there's a problem with the DV/PVC - the `Scheduling` phase is misleading, at least for me. 

@kbidarkar, wdyt?

Reply:
Initially I opened a CDI issue for that https://github.com/kubevirt/containerized-data-importer/issues/3028. Had a confirmation that `Scheduling` phase should also be fine.

> I would expect a Scheduling VMI to become scheduled at some point

As soon as the DV becomes available, it will be scheduled.

Reply:
Ok, let's proceed this way. Thanks!

Reply:
@jpeimer We discussed this and feel it should be ok.